gtsam/geometry/Similarity3.h

```diff
@@ -30,6 +30,8 @@ namespace gtsam {
 // Forward declarations
 class Pose3;
 
+typedef std::vector<Point3Pair> PointPairs;
```

Comment:
Keep this as `using Point3Pairs= vector<Point3Pair>`. C++11 compliant and preferred (as it was previously). :slightly_smiling_face:

Reply:
thanks for the review @varunagrawal. I'll make the change -- is there a specific reason `using` is preferred over `typedef`?

Reply:
https://stackoverflow.com/a/23196675/1236990

Reply:
If Pose3Pairs is defined in Pose3.h, Point3Pairs (indeed, rename it?) should be defined in Point3.h

Reply:
Updated